src/openrct2/world/MapAnimation.cpp

```diff
@@ -88,14 +87,14 @@ void map_animation_invalidate_all()
  *
  *  rct2: 0x00666670
  */
-static bool map_animation_invalidate_ride_entrance(int32_t x, int32_t y, int32_t baseZ)
+static bool map_animation_invalidate_ride_entrance(CoordsXYZ loc)
 {
-    auto tileElement = map_get_first_element_at(x >> 5, y >> 5);
+    auto tileElement = map_get_first_element_at(loc.x >> 5, loc.y >> 5);
```

Comment:
What I've been doing is creating a `auto tileLoc = TileCoordsXYZ{loc}`. Then all the shifting is implied and the code also is more clear on where you're using `loc` or `tileLoc`. I guess it's a good format/convention to stick to (and applies to most of the changes in this file)